### ISSUE TITLE
Document who the Council representatives are including important affiliation information

### ISSUE THREAD

**Issue by rylev** [rust-lang/leadership-council#9]
We want to make sure we have documented who the Council representatives are. This might be as simple as linking to the team repo and/or [governance page](https://www.rust-lang.org/governance/teams/leadership-council).

We also want to make sure we document important affiliation information. Documenting employer might be sufficient.

**Comment by jonathanpallant** [comment]
Can the existing rust-lang team system record this information?

**Comment by rylev** [comment]
No, but we could add it. However, we'd only really want this to apply to the Council, and it might send the wrong signals if we add this even as an optional field for others to add. 

It seems the pro of this information being in the team repo is that the team repo is the logical home for this (as it is supposed to be the canonical place for team related information), but the con is that this is fairly Council specific so it's a bit awkward to add.

**Comment by eholk** [comment]
Is there anything that still needs to be done here? We have the councilmembers listed on the web site, but I guess we don't have affiliation information listed there yet.

**Comment by jonathanpallant** [comment]
For what it's worth, for my time on the very lowest tier of elected office in the UK, I was required to post a public document listing all my financial interests, including my home address (because it was property in the ward that I owned). So I'm generally fine with asking people to make it clear where their financial interests lie - but putting it in the teams TOML file seems better than putting it on our Twitter feed.
